tests/Fixtures/AbstractResetPasswordTestKernel.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace SymfonyCasts\Bundle\ResetPassword\Tests\Fixtures;
+
+use Symfony\Bundle\FrameworkBundle\FrameworkBundle;
+use Symfony\Bundle\FrameworkBundle\Kernel\MicroKernelTrait;
+use Symfony\Bundle\FrameworkBundle\Routing\Loader\Configurator\RoutingConfigurator;
+use Symfony\Component\Config\Loader\LoaderInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\HttpKernel\Kernel;
+use SymfonyCasts\Bundle\ResetPassword\SymfonyCastsResetPasswordBundle;
+
+/**
+ * @author Jesse Rushlow <jr@rushlow.dev>
+ * @author Ryan Weaver   <ryan@symfonycasts.com>
+ *
+ * @internal
+ */
+class AbstractResetPasswordTestKernel extends Kernel
```

Comment:
Probably will be using this in the repository functional tests. If not && able, will embed this in the integration test itself.